NEWS.md

```diff
@@ -10,8 +10,14 @@ v1.16.0, Fri Sep 30, 2022
 =========================
 
 ## Core
+- Added CUDA IPC support
```

Comment:
I would rephrase it to `Added HMEM IPC cache mechanism`. CUDA IPC support is already introduced in earlier libfabric versions.

Reply:
>CUDA IPC support is already introduced in earlier libfabric versions.

I am not sure. Previous version had ZE IPC, and SAR protocol for CUDA, but does not have CUDA IPC

But the item "CUDA IPC support" should be in SHM provider's NEWs, not core. Core should have "Added HMEM IPC cache mechanism" as you suggested.

Reply:
Looks like the shm change has already been covered in @aingerson's PR. so I did not include it in this PR